control-box/PatternList.h

```diff
@@ -0,0 +1,40 @@
+#include "Pattern.h"
+#include <FastLED.h>
```

Comment:
I assume that FastLED already has an ifdef block of that kind within it.

Yes, I could just list it in PatternState.h (the root dependency)